docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -158,6 +161,78 @@ Not all packages are required to run Bareos.
+| Previous major releases | (Source code only)                                                           | `Subscription repository <https://download.bareos.com/bareos/release/>`_       |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| Current major release   | `Community repository <https://download.bareos.org/current/>`_               | `Subscription repository <https://download.bareos.com/bareos/release/>`_       |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| Development binaries    | https://download.bareos.org/next/                                                                                                                             |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+|  **Community**                                                                                                                                                                          |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| Mailing list	          | `bareos-users <https://groups.google.com/forum/#!forum/bareos-users>`_                                                                                        |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+|  **Support**                                                                                                                                                                            |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| without service level   |                                                                              | `available <https://www.bareos.com/support/>`_                                 |
++-------------------------+------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
+| with service level      |                                                                              | `available <https://www.bareos.com/support/>`_                                 |
```

Comment:
```suggestion
| with service level      |                                                                              | `available at www.bareos.com/support/ <https://www.bareos.com/support/>`_                                 |
```

Reply:
We should sync with sales about the wording here

Reply:
updated